feder/cases/filters.py

```diff
@@ -14,8 +14,8 @@
 class CaseFilter(DisabledWhenFilterSetMixin, django_filters.FilterSet):
     created = django_filters.DateRangeFilter(label=_("Creation date"))
     voivodeship = DisabledWhenVoivodeshipFilter()
-    county = DisabledWhenCountyFilter()
-    community = DisabledWhenCommunityFilter()
+    county = DisabledWhenCountyFilter(label=_("County"))
```

Comment:
Ta klasa `DisabledWhenCountyFilter` już dotyczy `County`, więc proponuje odpowiednie tłumaczenie zawrzeć w niej, zamiast przekazywać to niej. To oznacza zmianę tylko w jednym miejscu: https://github.com/watchdogpolska/feder/blob/master/feder/teryt/filters.py

Reply:
Nie wiem, czy dobrze zrozumiałam. Zaktualizowałam PR. Poprawkę zrobiłam w `DisabledWhenCountyFilter` i `DisabledWhenCommunityFilter`.